PrebidMobile/PrebidMobileRendering/Prebid/PBMCore/Prebid.swift

```diff
@@ -0,0 +1,165 @@
+/*   Copyright 2018-2021 Prebid.org, Inc.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import OMSDK_Prebidorg
+
+fileprivate let defaultTimeoutMillis = 2000
+
+@objcMembers
+public class Prebid: NSObject {
```

Comment:
let's move this file into some common place. For example, we can create a new folder in the root codebase directory - `ConfigurationAndTargeting` for this purpose. And move all respective files into this folder eventually.